lms/envs/common.py

```diff
@@ -5387,10 +5387,13 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 SUBSCRIPTIONS_TRIAL_LENGTH = 7
 SUBSCRIPTIONS_SERVICE_WORKER_USERNAME = 'subscriptions_worker'
 
-############## NOTIFICATIONS EXPIRY ##############
+############## NOTIFICATIONS ##############
 NOTIFICATIONS_EXPIRY = 60
 EXPIRED_NOTIFICATIONS_DELETE_BATCH_SIZE = 10000
 NOTIFICATION_CREATION_BATCH_SIZE = 99
+NOTIFICATIONS_DEFAULT_FROM_EMAIL = "no-reply@example.com"
+NOTIFICATION_TYPE_ICONS = {}
+DEFAULT_NOTIFICATION_ICON_URL = "https://edx-notifications-static.edx.org/icons/newspaper.png"
```

Comment:
Can we have a `NOTIFICATION_STATIC_ASSETS_ROOT_URL`, and extend that for this URL!

Something like `NOTIFICATION_STATIC_ASSETS_ROOT_URL/icons/newspaper.png`

Reply:
Good idea. Will do that